_data/documentsupdated.yaml

```diff
@@ -867,6 +867,9 @@
     - pagename: HTTP Response Codes
     - pagename: Configuration of Push Proxy
     - pagename: IP Whitelisting
+  - documentname: Proactive Messaging
```

Comment:
@lmartliveperson I have added the proactive messaging link in the navigation menu.